docs/contributing-concept.md

```diff
@@ -0,0 +1,142 @@
+
+<img height=1080 src="https://markdown-engineering.herokuapp.com/svg/graph LR;  Basics[Minimal compilable file] --> Independents[Import, Exposing, Functions, Float, Int, Type Annotations];  Independents --> Mathematical-Operators;  Independents --> Mathematical-Functions;  Independents --> Let-Expressions;  Independents --> Equality;  Equality --> Ordering;  Ordering --> Comparisons;  Independents --> String;  String --> Regex;  Independents --> Char;  Independents --> List;  List --> List-Extra;  Independents --> Dict;  Independents --> Set;  Independents --> Array;  Independents --> Type-alias;  Independents --> Sum-types;  Independents --> Booleans[Bool type & operators, If];  Sum-types --> Pattern-matching;  Pattern-matching --> Maybe;  Booleans --> Maybe;  Maybe --> Result;  Pattern-matching --> Parser;  Independents --> Function-composition;  Independents --> Function-chaining;  Independents --> Partial-application;  Partial-application --> Point-freestyle;  Function-composition --> Point-freestyle;  Function-chaining --> Point-freestyle;  Point-freestyle --> Operator-functions;  Independents --> Bitwise-operators;  Independents --> Conversions;  Independents --> Posix-time"/>
+
+Identify a missing concept that you want to implement.
+Beware that the graph above is a moving target and may change as we find that prerequisites should differ.
+
+## 2. Define the learning goals
+
+It is important to know what a concept exercise aims to teach and what is out of scope so let's create our first file: `exercises/concept/<story-name>/.meta/design.md`.
+Since concept exercises are named after the story they use we don't know it yet, so we'll keep `story-name` for now and replace it later.
+The [`design.md` specification][design-spec] states that this document should define the overall goal of the exercise, its multiple learning objectives, the things that are out of scope, the associated concept it unlocks and the concepts prerequisites to be able to complete the exercise.
+You can use the [`design.md` of the boolean concept exercise][design-bool] as a template starting point.
+
+## 3. Write a concept introduction for the exercise
+
+Once the learning objectives are clear, it's time to write an introduction explaining enough of the concept for a student to then solve the exercise.
+Create a file `exercises/concept/<story-name>/.docs/introduction.md`.
+The detailed contents of this file are explained in [its specification][exercise-intro-spec].
+You can use the [`introduction.md` of the boolean concept exercise][intro-bool] as a template starting point.
+
+The information provided should introduce the concept just enough for the student to figure out the exercise solution themselves.
+Use proper technical terms to ease student online search but try to avoid links and to explain things concisely inline in the document.
```

Comment:
```suggestion
Use proper technical terms to ease student online search but try to avoid doing all the work for them (for example, by providing all the necessary links or exhaustively explaining everything).
```

Reply:
My sentence was not clear. What I meant was that there is an emphasis on explaining things inline instead of making students click on links and leave the page. Would this have been clearer?
> try to avoid links and try to explain things concisely inline.

Reply:
maybe "try to avoid links and instead explain things concisely inline" ?

Reply:
yeah that's better!